bentoml/_internal/models/model.py

```diff
@@ -27,6 +27,7 @@
 from ..store import Store
 from ..store import StoreItem
 from ..types import MetadataDict
+from ..types import MetadataType  # type: ignore
```

Comment:
Why's this needed? I think it would be nice for every type ignore to have a short blurb describing why it's there.

Reply:
otherwise every bentoml commands won't work